src/sentry/integrations/pipeline.py

```diff
@@ -126,10 +126,10 @@ def _finish_pipeline(self, data):
                             return self._dialog_response(
                                 {
                                     "error": _(
-                                        "The provided %s account is linked to a different Sentry user. "
-                                        "To continue linking the current Sentry user, please use a different %s account."
+                                        "The provided %(proper_name)s account is linked to a different Sentry user. "
```

Comment:
Fixes a warning, it's preferable to use named args because of how different languages work so that you can reference them (it'd be more applicable if these were different).

Reply:
That is not applicable with translations iirc.